src/company/HomeWorkAppLessen2.java

```diff
@@ -0,0 +1,40 @@
+package company;
+
+public class HomeWorkAppLessen2 {
+    public HomeWorkAppLessen2() {
+    }
+
+    public static void main(String[] args) {
+        System.out.println(comparison(6, 5));
+        integer(-1);
+        System.out.println(compareTheNumbers(-1));
+        rowToColumn("System.out.println", 5);
+    }
+
+    public static boolean comparison(int a, int b) {
+        boolean isComparison = a + b >= 10 && a + b <= 20;
+        return isComparison;
```

Comment:
поскольку логическое выражение само по себе возвращает boolean результат, то можно упростить:
```suggestion
        return a + b >= 10 && a + b <= 20;
```